### DISCUSSION TITLE
how to open .png pictures with mpv?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14218] [Q&A]
Hi, opening .jpeg works with mpv but when I try to drag and drop a .png file on mpv, there is only a black screen. So I don't know if there is something you need to put into your config file to make it work, or isn't .png supported?

Thank you :)

**Comment by kasper93** [comment]
Where did you get mpv build? Is png decoder enabled in ffmpeg?

**Reply by geextahslex** [reply]
>Where did you get mpv build?

I always get it from https://sourceforge.net/projects/mpv-player-windows/files/
>Is png decoder enabled in ffmpeg?

I don't know ^^'
